src/ui/main_window.ui

```diff
@@ -340,7 +340,7 @@
   </action>
   <action name="actionPackage_Exporter">
    <property name="text">
-    <string>Package exporter (experimental)</string>
+    <string>Package exporter</string>
```

Comment:
To be honest that name is on there because it was too complicated to use and had too much text for people to read before they could use it. I have a proposal to improve that in the PR I made...